andela/titan-dev-ah-frontend
# #165054820 Setup Test Env

#### What does this PR do?
- Configures test environment 
#### Description of Task to be completed?
- Configures jest and enzyme test env
- Add sample tests
- Add test coverage
#### How should this be manually tested?
- Fork the repo and check out the branch `ch-setup-test-165054820`
- In the root folder run `yarn` to install all dependencies
- Run `yarn test` to see if test are working
- Check your terminal to see the test coverage

#### What are the relevant pivotal tracker stories?
#165054820
